nncf/quantization/algorithms/post_training/openvino_backend.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import List
+
+import openvino.runtime as ov
+
+from nncf.common.graph.graph import NNCFNode
+from nncf.common.graph.transformations.commands import TargetType
+from nncf.openvino.graph.metatypes.openvino_metatypes import OVIfMetatype
+from nncf.openvino.graph.transformations.commands import OVExtractIfSubgraphCommand
+from nncf.openvino.graph.transformations.commands import OVOutputInsertionCommand
+from nncf.openvino.graph.transformations.commands import OVTargetPoint
+from nncf.openvino.graph.transformations.commands import OVUpdateIfSubgraphCommand
+from nncf.quantization.algorithms.post_training.backend import PostTrainingBackend
+
+
+class OVPostTrainingBackend(PostTrainingBackend):
+    IF_OP_MODEL_INPUT_PORTS = (0, 1)
```

Comment:
Since we provide the `IF_OP_MODEL_INPUT_PORTS` variable, we should use it instead/or inside methods.

Reply:
Removed